src/Microsoft.ML.LightGBM/LightGbmArguments.cs

```diff
@@ -401,7 +406,7 @@ public enum EvalMetricType
             }
             if (!string.IsNullOrEmpty(metric))
                 res["metric"] = metric;
-            res["sigmoid"] = "0.5";
+            res["sigmoid"] = Sigmoid.ToString();
```

Comment:
>.ToString(); [](start = 36, length = 12)

Please, no ToString for this parameter, and any other in this method (except EvalMetric), also please remove, ToString from UpdateParameters in LightGbmArguments.cs  in field.GetValue(Args).ToString()